cloud/scope/managed_control_plane.go

```diff
@@ -216,6 +219,22 @@ func (s *ManagedControlPlaneScope) GetOrCreateControlPlane(ctx context.Context)
 	return s.getOKEClusterFromOCID(ctx, clusterId)
 }
 
+func getOKEClusterTypeFromSpecType(controlPlaneSpec infrav2exp.OCIManagedControlPlaneSpec) oke.ClusterTypeEnum {
+	if controlPlaneSpec.ClusterType != "" {
+		switch controlPlaneSpec.ClusterType {
+		case infrav2exp.BasicClusterType:
+			return oke.ClusterTypeBasicCluster
+			break
+		case infrav2exp.EnhancedClusterType:
+			return oke.ClusterTypeEnhancedCluster
+			break
+		default:
+			break
+		}
+	}
+	return ""
```

Comment:
Thought: don't have to change but thought about having an `oke.ClusterTypeUnknown` instead of `""` 🤷

Reply:
OKE, is the OKE API, currently ClusterTypeUnknown is no  a known type.  The code is such that if a cluster type is not mentioned, it should use empty which will use whatever OKE sets as default.